crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -577,47 +1515,54 @@ mod YASPoolTests {
     use yas_core::contracts::yas_router::{
         YASRouter, IYASRouterDispatcher, IYASRouterDispatcherTrait
     };
-    use yas_core::tests::utils::constants::PoolConstants::{TOKEN_A, TOKEN_B, POOL_ADDRESS, WALLET};
+    use yas_core::tests::utils::constants::PoolConstants::{
+        TOKEN_A, TOKEN_B, POOL_ADDRESS, WALLET, encode_price_sqrt_1_1, encode_price_sqrt_1_10
+    };
     use yas_core::tests::utils::constants::FactoryConstants::{
         POOL_CLASS_HASH, FeeAmount, fee_amount, tick_spacing
     };
     use yas_core::contracts::yas_erc20::{
         ERC20, ERC20::ERC20Impl, IERC20Dispatcher, IERC20DispatcherTrait
     };
 
-    fn setup() -> (IYASPoolDispatcher, IERC20Dispatcher, IERC20Dispatcher) {
+
+    fn setup() -> (
+        IYASPoolDispatcher, IERC20Dispatcher, IERC20Dispatcher, IYASRouterDispatcher, i32, i32
+    ) {
         let mint_callback = deploy_mint_callback(); // 0x1
         let yas_factory = deploy_factory(OWNER(), POOL_CLASS_HASH()); // 0x2
 
         // Deploy ERC20 tokens with factory address
-        let token_0 = deploy_erc20('YAS0', '$YAS0', 4000000000000000000, OWNER()); // 0x3
-        let token_1 = deploy_erc20('YAS1', '$YAS1', 4000000000000000000, OWNER()); // 0x4
+        let token_0 = deploy_erc20(
+            'YAS0', '$YAS0', 400000000000000000000000000000000000, OWNER()
+        ); // 0x3
+        let token_1 = deploy_erc20(
+            'YAS1', '$YAS1', 400000000000000000000000000000000000, OWNER()
+        ); // 0x4
 
         set_contract_address(OWNER());
-        token_0.transfer(WALLET(), 4000000000000000000);
-        token_1.transfer(WALLET(), 4000000000000000000);
+        token_0.transfer(WALLET(), 400000000000000000000000000000000000);
```

Comment:
f the tests don't require a specific value for tokens, could we directly set it to the max? What do you think?

Reply:
done